docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,156 @@
+---
+title: NuGet 5.0 RTM Release Notes
+description: Release notes for NuGet 5.0 including known issues, bug fixes, new features, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 04/02/2019
+ms.topic: conceptual
+---
+
+# NuGet 5.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.0.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.0](https://visualstudio.microsoft.com/downloads/) | 2.1.602<sup>1</sup>, 2.2.202<sup>2</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload 
+
+<sup>2</sup>Available as an optional install with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.0
+
+* Support solution filter when restoring the solution via CLI or in Visual Studio - [#5820](https://github.com/NuGet/Home/issues/5820)
```

Comment:
Just changed bug title to:
Support VS 2019 Solution Filter feature. Also supports project not in solution, or unloaded projects. Need to restore complete solution (via CLI or VS) first.

@karann-msft  - Can you use that same text (or similar) in the summary?

Also, can you please manually change the note about that fix below to use that new title?